tests/samplers_tests/tpe_tests/test_parzen_estimator.py

```diff
@@ -250,7 +228,9 @@ def test_calculate_shape_check_categorical(
     mpe = _ParzenEstimator(
         {"c": mus}, {"c": distributions.CategoricalDistribution([0.0, 1.0, 2.0])}, parameters
     )
-    assert len(mpe._mixture_distribution.weights) == max(len(mus) + int(prior), 1)
+    assert len(mpe._mixture_distribution.weights) == max(
+        len(mus) + 1, 1
```

Comment:
Same here. We don't need the max operator.
```suggestion
        len(mus) + 1
```